invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/components/ResultsList/ResultsList.js

```diff
@@ -1,68 +1,47 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Table, Button } from 'semantic-ui-react';
+import { document as documentApi } from '../../../../../common/api';
+import { fromISO, toString } from '../../../../../common/api/date';
+import { openRecordEditor } from '../../../../../common/urls';
+import { ResultsTable } from '../../../../../common/components';
+import { NewButton } from '../../../components/buttons';
 
-class DocumentRow extends Component {
+export class ResultsList extends Component {
   constructor(props) {
     super(props);
-    this.viewDetailsClickHandler = props.viewDetailsClickHandler;
+    this.viewDetailsClickHandler = this.props.viewDetailsClickHandler;
   }
 
-  render() {
-    const DocumentRecord = this.props.DocumentRecord;
-    const Document = DocumentRecord.metadata;
-
-    return (
-      <Table.Row key={DocumentRecord.id} data-test={DocumentRecord.id}>
-        <Table.Cell textAlign="center">
-          <Button
-            circular
-            compact
-            icon="eye"
-            onClick={() => {
-              this.viewDetailsClickHandler(DocumentRecord.id);
-            }}
-          />
-        </Table.Cell>
-        <Table.Cell>{DocumentRecord.id}</Table.Cell>
-        <Table.Cell data-test="mapped-status">{Document.title}</Table.Cell>
-        <Table.Cell>{Document.authors}</Table.Cell>
-        <Table.Cell>{`${Document.abstracts[0].substring(
-          0,
-          50
-        )}...`}</Table.Cell>
-      </Table.Row>
-    );
-  }
-}
+  _getFormattedDate = d => (d ? toString(fromISO(d)) : '');
```

Comment:
I have added the function in date.js to avoid adding this line to components

Reply:
Ok thanks, I will check and replace.